<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="vlTop">
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Traverse</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlMapGen">
        <item>
         <spacer name="spacerA">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblMapSize">
          <property name="text">
           <string>Map Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboMapSize">
          <item>
           <property name="text">
            <string>Small</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Large</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="spacerB">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblMapType">
          <property name="text">
           <string>Map Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboMapType">
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Volcano</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Oasis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Factory</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Forest</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="spacerC">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnGenerate">
          <property name="text">
           <string>Generate New Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnImport">
          <property name="text">
           <string>Import Map</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hlMapCode">
        <item>
         <widget class="QLabel" name="lblMapCode">
          <property name="text">
           <string>Map Generation Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMapCode">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCopyMapCode">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vlCenter">
      <item>
       <layout class="QGridLayout" name="glMap"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vlBottom">
      <item>
       <widget class="QLabel" name="lblScore">
        <property name="text">
         <string>Score: 0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtConsole">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearConsole">
        <property name="text">
         <string>Clear Console</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
